src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -125,7 +126,7 @@ def __init__(  # pylint: disable=too-many-arguments
             resampler_config: The resampler configuration in case resampling is required.
             window_alignment: A datetime object that defines a point in time to which
                 the window is aligned to modulo window size.
-                (default is midnight 01.01.01)
+                (default is midnight 01.01.01 UTC)
```

Comment:
I know this is not introduced by this PR, but since you are touching it, I would update this to use ISO format for dates (0001-01-01T00:00:00+00:00), is not that it is super clear, but using this German short format is **very** confusing.

Reply:
will update it as suggested.

Reply:
My Bad! Sorry about that ;).

Reply:
I fixed it in a separate commit